### PR TITLE
Convert GitHub-Based Notifications to E-Mail

### PR DESCRIPTION
**Pull request by Noordfrees**
Instead of using an issue (#31) as a noticeboard, send all sorts of messages via e-mail to those administrators who wish to receive them. Anyone in @widelands/chieftains can add themselves to the database (see the README change, or ask me to add you) if you want to receive server notices.

And a bit more cleanup.

### CODE REVIEW

File: wl/server/TransifexIntegration.java [hunk 1]
```diff
@@ -223,39 +222,30 @@ public synchronized void checkIssues() throws Exception {
...
-					    "\n  String ID     : " + i.stringID + "\n  Occurrences   : " +
-					    i.occurrence + "\n  Last modified : " + i.datetime_modified +
-					    "\n  Priority      : " + i.priority + "\n  Issue message : " + i.message);
+					    // anti-linebreak comment
+					    + "\n  Issue ID      : " + i.issueID            // anti-linebreak comment
+					    + "\n  Source String : " + i.string             // anti-linebreak comment
+					    + "\n  String ID     : " + i.stringID           // anti-linebreak comment
+					    + "\n  Occurrences   : " + i.occurrence         // anti-linebreak comment
+					    + "\n  Last modified : " + i.datetime_modified  // anti-linebreak comment
+					    + "\n  Priority      : " + i.priority           // anti-linebreak comment
```
Comment (matthiakl) [inline wl/server/TransifexIntegration.java:242]: `// anti-linebreak comment` :+1:

File: wl/utils/Utils.java [hunk 2]
```diff
@@ -648,60 +652,48 @@ public static long filesize(File dir) {
...
+		File message = Files.createTempFile(null, null).toFile();
+		PrintWriter write = new PrintWriter(message);
+		write.println("From: noreply@widelands.org");
+		write.println("Subject: " + subject);
+		write.println();
+		write.println(body);
+		write.print(
+		    "\n-------------------------\n"
+		    +
+		    "To change how you receive notifications, please go to https://www.widelands.org/notification/.");
```
Comment (matthiakl) [inline wl/utils/Utils.java:671]: Should this PR wait until the website notification settings are available?

Comment (Noordfrees) [reply wl/utils/Utils.java]: No need IMHO, and the string issue mails already do contain this note… I did however add a switch to suppress the footer in the new admin mails as those bypass the notification settings.

**Review by matthiakl — APPROVED** [review]

**Comment by Noordfrees** [comment]
Deployed. Thanks for the review

**Pull request merged**